### PR TITLE
Clarify source of API docs

### PR DESCRIPTION
**Pull request by TheEnbyperor**
Signed-off-by: Q <q@as207960.net>

Arose whilst trying to fix issue #237

### CODE REVIEW

File: content/en/docs/reference/.where-are-the-docs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# AAAAA?! Where be the docs?
+
+API Docs are autogenerated from the Golang source.
+Go edit the source comments [here](https://github.com/jetstack/cert-manager/tree/master/pkg/internal/apis) to update API docs.
```
Comment (meyskens) [inline content/en/docs/reference/.where-are-the-docs:4]: Can we make this the versioned API docs as we document those and not the internal one, which is only used for conversion logic.

Comment (maelvls) [reply content/en/docs/reference/.where-are-the-docs:4]: @TheEnbyperor Here is the link you can use:
```suggestion
Go edit the source comments [here](https://github.com/jetstack/cert-manager/tree/master/pkg/apis) to update API docs.
```

**Comment by SgtCoDFish** [comment]
Wow, I'd not seen this before and I _really_ like the idea of adding this. I was caught out by this more than once when I started contributing to cert-manager!

Unfortunately we've got a new website refresh coming up and since this PR has been open for so long (sorry about that) I don't think it'll make sense in the refreshed version (and I'd actually like to always check in the generated docs just so there's _something_ in the repo).

As such I'm going to close this, but thanks again @TheEnbyperor  - really great idea and I love the thought behind it!

**Pull request closed without merging**